src/test/java/com/apicatalog/jsonld/benchmark/README.md

```diff
@@ -0,0 +1,13 @@
+
+
+## Original results
+```
+Benchmark                     Mode  Cnt     Score     Error  Units
+LoadingBenchmark.toRdfApiGet  avgt    5  5859.114 ± 522.500  ms/op
+```
+
+## Current results
+```
+Benchmark                     Mode  Cnt     Score     Error  Units
+LoadingBenchmark.toRdfApiGet  avgt    5  3363.833 ± 448.450  ms/op
+```
```

Comment:
Currently I'm at close to a 2x performance improvement.

Reply:
The improvement is great! I'm afraid that without a reference computing environment  (cpu, mem, etc) the numbers are  not helpful to some who has just visited the project page. Please note that after releasing this PR the  table for Previous version becomes obsoleted. 

Not sure if is that doable, but would be great to somehow use github actions and virtual machines (as  a reference environment) to generate such report for each PR.

Reply:
Yeah, the results depend heavily on what computer you're using. The file is nice for tracking performance of this branch, but I'll delete it before merging.

I would love to have a github action or similar to run the benchmarks with a before/after for each PR. I've not found anything that does this yet.

Reply:
Perhaps, there is no need to run an action before, we can keep benchmark results for the main branch and use it for comparison. I've not found anything easy to use yet as well.